wireless_modules/mqtt_test/boot.py

```diff
@@ -0,0 +1,34 @@
+import config 
```

Comment:
I was waiting for you to write this @nix-chu

Reply:
Just a thought, should this be a json or .env config file rather than an actual python file? Not sure if that'll work in micropython though

Reply:
> I was waiting for you to write this @nix-chu

@khanguslee config file or a readme?

Reply:
I believe @Blake-Haydon said `.env` files don't work in micropython. IDM a JSON file as well?

@nix-chu Oh I meant the suggestion since you said the same thing in my PR. 😄

Reply:
I think actual python is good, as it is handy to initiate sensors and LEDs in this file.

Reply:
> Any chance we can catch an error if config.py does not exist and/or cannot be found? Only asking because it will happen and people will ask you how to fix it!

I have added @angustrau build script so hopefully that doesn't happen

Reply:
Oops my bad, I read it wrong, will add in the try catch now.